al2/rust/hello/{{ cookiecutter.project_slug }}/template.yaml

```diff
@@ -20,9 +20,11 @@ Resources:
 
   PutFunction:
     Type: AWS::Serverless::Function # More info about Function Resource: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#awsserverlessfunction
+    Metadata:
+      BuildMethod: rustcargolambda # More info about Cargo Lambda: https://github.com/cargo-lambda/cargo-lambda
```

Comment:
Let's don't forget to update this if we decide the change the name of the build method